python/helper-image/launcher/cmd.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2021 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"os"
+	"os/exec"
+
+	"github.com/sirupsen/logrus"
+)
+
+// for testing
+var newCommand = createCommand
+var newConsoleCommand = createConsoleCommand
+
+// commander is a subset of exec.Cmd
+type commander interface {
+	Run() error
+	Output() ([]byte, error)
+	CombinedOutput() ([]byte, error)
+}
+
+var _ commander = (*exec.Cmd)(nil)
```

Comment:
nit: might be helpful to add a comment that this is check verifying that exec.Cmd satisfies the `commander` interface